load/UnicodeScript.py

```diff
@@ -28,13 +28,14 @@ def getFilenames(self, s3Client, location, filesetPath):
 				self.errors.append("ERROR: Invalid pathname %s" % (pathname,))
 		else:
 			bucket = location[5:]
-			#print("bucket", bucket)
-			request = { 'Bucket': bucket, 'MaxKeys': 1000, 'Prefix': filesetPath + "/" }
+			request = { 'Bucket': bucket, 'MaxKeys': 1000, 'Prefix': "from-text-processing/" + filesetPath + "/" }
```

Comment:
the Prefix "from-text-processing" is for development only. I think I will rearrange the content in the S3 bucket to move it up one level, so there is no need for the "from-text-processing" prefix

Reply:
@bradflood, the Prefix "from-text-processing" has been removed from entire project.